tests/catboost/test_catboost_model_export.py

```diff
@@ -391,6 +391,7 @@ def test_pyfunc_serve_and_score(reg_model):
         model_uri,
         data=inference_dataframe,
         content_type=pyfunc_scoring_server.CONTENT_TYPE_JSON_SPLIT_ORIENTED,
+        extra_args=EXTRA_PYFUNC_SERVING_TEST_ARGS,
```

Comment:
`EXTRA_PYFUNC_SERVING_TEST_ARGS` looks like this:
https://github.com/mlflow/mlflow/blob/9c934f5a57cc8701475eb7ed0b6b072600b66bed/tests/catboost/test_catboost_model_export.py#L33